src/components/inline/Button/Button.js

```diff
@@ -3,18 +3,59 @@ import React from "react";
 
 import s from "./Button.module.css";
 
-const Button = ({ children }) => (
-  <button className={s.button} type="button">
-    {children}
-  </button>
-);
+const Button = ({ button }) => {
+  let buttonView;
 
-Button.propTypes = {
-  children: PropTypes.node,
+  if (button.externalLink) {
+    buttonView = (
+      <a rel="noreferrer" target="_blank" href={button.externalLink}>
+        <button type="button" className={s.button}>
+          {button.text}
+        </button>
+      </a>
+    );
+  } else if (button.internalLink) {
+    buttonView = (
+      <a href={button.internalLink}>
+        <button type="button" className={s.button}>
+          {button.text}
+        </button>
+      </a>
+    );
+  } else {
+    buttonView = (
+      <button
+        type="button"
+        className={s.button}
+        onClick={() => {
+          button.scriptAction();
+        }}
+      >
+        {button.text}
+      </button>
+    );
+  }
+
+  return <div>{buttonView}</div>;
```

Comment:
You should remove this extra `<div>`, since it causes the component to become a block element, not an inline element.

Also, you may find it easier to just `return` from each of the inner conditional cases, since there's no need to assign them to a variable only to just directly return it in the end.